tools/gce_setup/grpc_docker.sh

```diff
@@ -428,6 +428,7 @@ grpc_interop_test_args() {
       python)       grpc_port=8050 ;;
       ruby)         grpc_port=8060 ;;
       csharp_mono)  grpc_port=8070 ;;
+      csharp_dotnet) grpc_port=8070 ;;
```

Comment:
I think we do. They will never run on the same machine and both csharp_mono and csharp_dotnet is still csharp afterall.